data/src/Cardano/Data/StrictFingerTree.hs

```diff
@@ -41,13 +43,15 @@ module Cardano.Data.StrictFingerTree
 
     -- ** Maps
   , fmap'
+  , unsafeFmap
 
   -- Re-export from "Data.FingerTree"
   , Measured(..)
   , ViewL (..)
   , ViewR (..)
   ) where
 
+import           Cardano.Data.StrictUnit(forceElemsToWHNF)
```

Comment:
Missing space between `..Unit` and `(`.

Nitpicking: I would move this import down so that it is no longer grouped with `Prelude`, but a new group at the bottom. I believe it's a convention to group imports from the local repo or package at the bottom. So `Prelude` imports first, then a group with 3rd-party imports, then a group with local imports.

Reply:
done